source/about_jena/security-advisories.md

```diff
@@ -50,9 +50,23 @@ policy above we advise users to always utilise the latest Jena release available
 
 Please refer to the individual CVE links for further details and mitigations.
 
+## CVE-2023-22665 - Exposure of arbitrary execution in script engine expressions.
+
+[CVE-2023](https://www.cve.org/CVERecord?id=CVE-2023-22665) affects Jena ?? through 4.7.0 and relates to the [Javascript
```

Comment:
- [ ] Need to go back through release notes to see when this feature was introduced

Reply:
Feature was introduced in Jena 3.7.0